client/src/pages/DetailsPage/item-details.module.scss

```
@@ -2,26 +2,179 @@
+            color: $fc-4;
+        }
     }
 
-    .quantitySelection {
-        font-family: $ff-roboto;
-        font-size: 5rem;
-        background-color: green;
+    hr {
+        color: #c0c0c0;
     }
 
+    .quantitySection {
+        margin-top: 4%;
+
+        .quantityTitle {
+            font-size: $fz-small-4;
+            color: $fc-4;
+            margin-bottom: 3%;
+        }
+
+        .selectQuantity {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            margin-bottom: 6%;
+
+            .minusQuantity,
+            .plusQuantity {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                font-size: $fz-large-1;
+                padding: 5.4%;
+                width: 2rem;
+                height: 2rem;
+                border-radius: 50%;
+                border-color: transparent;
+
+                &:hover {
+                    background-color: #DEFFE4;
+                    color: $fc-3;
+                    transform: scale(1.1);
+                    transition: transform 0.4s;
+                }
+
+                &:active {
+                    background-color: $bc-1;
+                    box-shadow: 0 0 5px #DEFFE4, 0 0 10px #DEFFE4, 0 0 20px #DEFFE4, 0 0 30px #DEFFE4;
+                    color: $fc-2;
+                    font-weight: 500;
+                }
+            }
+
+            .numberQuantity {
+                font-size: $fz-large-1;
+                margin: 0 16%;
+            }
+        }
+
+        .addCartButton {
+            font-size: $fz-small-5;
+            width: 80%;
+            height: 7vh;
+            margin-bottom: 7.5%;
+            background-color: $bc-2;
+            border-color: transparent;
+            border-radius: 10px;
+
+            &:hover {
+                background-color: #DEFFE4;
+                color: $fc-3;
+                transform: scale(1.01);
+                transition: transform 0.4s;
+                font-weight: 400;
+            }
+
+            &:active {
+                background-color: $bc-1;
+                box-shadow: 0 0 5px #DEFFE4, 0 0 10px #DEFFE4, 0 0 20px #DEFFE4, 0 0 30px #DEFFE4;
+                color: $fc-2;
+                font-weight: 600;
+            }
+        }
+
+        .shoppingSection {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+
+            img {
+                width: 7%;
+                margin-bottom: 4%;
+            }
+
+            p {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                background-color: $bc-1;
+                color: $fc-2;
+                transform: translate(96%, -230%);
+                font-size: $fz-small-3;
+                padding: 2.4%;
+                width: 2rem;
+                height: 2rem;
+                border-radius: 50%;
```

Das ist soweit erstmal okay. Wenn du hier aber einen button hast der bisschen länger ist wird der Oval gestreckt. Das kannst du unterbinden indem du einfach einen sehr hohen Wert für Border-Radius nimmst.
```suggestion
                border-radius: 999vw;
```